reproducibility_project/src/engine_input/mcccs/fort77maker_twobox.py

```diff
@@ -0,0 +1,457 @@
+"""Takes an mbuild filled_box and creates a restart file for mcccs simulations."""
+import errno
+import os
+import sys
+from collections import Counter
+from subprocess import call
+
+import mbuild as mb
+import numpy as np
+
+print(sys.executable)
+
+
+def unique(list1):
+    """Delete duplicate elements in a list.
+
+    Parameters
+    ----------
+    list1 : Input list that may contain duplicate elements
+
+    Returns
+    -------
+    unique_list : Output list with duplicate elements deleted
+
+    """
+    unique_list = []
+
+    # traverse for all elements
+    for x in list1:
+        # check if exists in unique_list or not
+        if x not in unique_list:
+            unique_list.append(x)
+    # print list
+    return unique_list
+
+
+# This function takes name_list, coordinates, and writes the xyz file in out_filename
+def xyzwriter(name_list, coordinates, out_filename):
+    """Write an xyz file from element names and coordinates.
+
+    Parameters
+    ----------
+    name_list : List containing element names
+    coordinates : xyz coordinates of elements in AA
+    out_filename : Name of the output filename
+
+    Returns
+    -------
+    NA
+    """
+    if not (len(name_list) == coordinates.shape[0]):
+        print(
+            "Error in xyzwriter. The atom list size and coordinate size is not same"
+        )
+        return
```

Comment:
```suggestion
        raise IndexError("The atom and coordinate lengths are different.")
```
Might be better to raise an error here instead of printing and returning as normal. I'm not sure if index error is the best choice, but it is my first thought.

Reply:
Added.